### DISCUSSION TITLE
launch argument that disables console/logging?

### DISCUSSION THREAD

**Discussion by Lreaper** [open-goal/jak-project#3410] [General]
Is there any way to launch Jak1 directly without spawning the log window? I'm currently launching the game without using the launcher with `--game jak1 -- -boot -fakeiso` parameters, but unfortunately the log window creates some issues when I launch the game through a front-end (bigbox in this case)

**Comment by Zedb0T** [comment]
https://learn.microsoft.com/en-us/windows/win32/procthread/process-creation-flags
![image](https://github.com/user-attachments/assets/73e10f7b-883d-4989-b76c-f6768b4b4d23)
